.vscode/launch.json

```diff
@@ -18,6 +18,22 @@
         "0"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug Current Test File",
+      "program": "${workspaceRoot}/packages/build/node_modules/.bin/_mocha",
+      "cwd": "${workspaceRoot}",
+      "autoAttachChildProcesses": true,
+      "args": [
+        "--config",
+        "${workspaceRoot}/packages/build/config/.mocharc.json",
+        "-t",
+        "0",
+        "$(node ${workspaceRoot}/packages/build/bin/get-dist-file ${file})"
```

Comment:
I assume this won't work on Windows, right?

Reply:
This has been bugging me, so I made a few changes:
- "Run Mocha tests" now has the correct windows-specific configuration.
- replaced the "Debug Current Test File" subshell by writing a program that invokes the mocha cli directly.
- removed the `autoAttachChildProcess` setting.
- confirmed I could set a breakpoint in `packages/metadata/src/decorator-factory.ts` and vscode would stop on it when "Debug Current Test File" is run on `packages/metadata/src/__tests__/unit/decorator-factory.unit.ts`